_episodes/10-shielding.md

```diff
@@ -0,0 +1,170 @@
+---
+title: "Live Coding Shielding Problem"
+teaching: 0
+exercises: 90
+questions:
+- "How can I perform a shielding calculation using Python?"
+keypoints:
+- "Use Python to write a simple shielding calculation script."
+- "Learn some best practices when writing Python."
+---
+## Shielding Problem 
+
+In this lesson we'll be doing a live coding exercise in which we calculate the
+uncollided flux on the other side of a heterogeneous shield composed of lead,
+water, and tungsten. We'll be performing this calculation as a 1-D problem using
+the following model:
+
+$$ \huge \Phi = \Phi_{o} e^{-\mu x} $$
+
+where 
+
+ - $$ \Phi_{o} $$ - is the initial flux at the front of the shield (units: $$ cm^{-2}s^{-1} $$)
+ - $$ \mu      $$ - is the attenuation coefficient of the shield (units: $$ cm^{-1} $$)
+ - $$  x       $$ - is the shield thickness (units: $$ cm $$)
+ - $$ \Phi     $$ - is the uncollided flux at the back of the shield (units: $$ cm^{-2}s^{-1} $$)
+
+To begin, with a simple script which defines these variables and performs
+the calculation for a shield with one material, lead, with a thickness of 15 cm.
+
+
+~~~
+{% include checkpoints/checkpoint0.py %}
```

Comment:
Is there anyway we can make these checkpoints expandable (kind of like other exercises)? This way students can try it and check if they are right so that if they happen to be following along, they don't just view the answer right away.

Reply:
This also makes me think we should pose the part before that as a an actual problem and supply the variable values. What do you think about that for all of these?

Reply:
Yeah I like that idea. I think I can nest the direct link to the script inside of the exercise as well. 

- [ ] Wrap scripts into collapsible exercise blocks